### ISSUE TITLE
Bug: this dae model can not parse

### ISSUE THREAD

**Issue by lucasjinreal** [assimp/assimp#4551]
macOS preview can normally vis this dae model:
[ybot.dae.zip](https://github.com/assimp/assimp/files/8793399/ybot.dae.zip)

but ASSIMP would load failed:

```
ERROR::ASSIMP:: Expected different index count in <p> element.
```

Please take a look?

**Comment by kimkulling** [comment]
I have just checked the file. With the latest master, I was able to reproduce your issue. Blender will load the model correctly. So we made a mistake ...

**Comment by kimkulling** [comment]
Found the issue: the expected count of indices for the model is not the count found in the file. I do not know the reason.

**Comment by jinfagang** [comment]
@kimkulling is there a way to pass it? then why both macOS preview and blender can load it

**Comment by kimkulling** [comment]
Right, that seems to be the reason. I don't have a clue why the poly-definition doesn't work right now. I guess there is an edge-case that was not implemented in Assimp.

**Comment by jinfagang** [comment]
I think we can pass that probematic node, truncated better than none. Doesn't it. : )

**Comment by enginmanap** [comment]
this was a warning on old releases, but not on the latest release, just as a side note.

**Comment by patrikpatrik** [comment]
has this been fixed yet? Im still getting this on dancing_vampire.dae for learnOpenGL tutorial example

**Comment by kimkulling** [comment]
No, I haven't found a good solution until now. The issue is still under investigation.

**Comment by enginmanap** [comment]
@kimkulling, it is fixed by my PR. this issue is just a whitespace parsing error. Please check out my PR.

**Comment by patrikpatrik** [comment]
@kimkulling @enginmanap, please check out bug issue #4659. I've managed to pinpoint exactly when dae collada stopped working. The last build that worked was Build 5.0.1 Bugfix release dated January 12, 2020. The very next build 5.1.0 release candidate 1 released on October 19, 2021 broke the dae loader.

This is bug can be reproduced using Joey De Vries LearnOpenGL on Skeletal Animation:
https://learnopengl.com/Guest-Articles/2020/Skeletal-Animation

**Comment by enginmanap** [comment]
![Screenshot](https://user-images.githubusercontent.com/237148/181037880-213dc049-48af-4239-93d9-c8573b762718.png)
with my patch, this model loads as expected.

**Comment by patrikpatrik** [comment]
@ enginmanap, Very nice! Do you have a direct link for your pull request? Sorry I've never done this before and wanted this working so bad! Thank you!

**Comment by enginmanap** [comment]
https://github.com/assimp/assimp/pull/4632

**Comment by JG-Adams** [comment]
It's working for me too! Thanks guys! :)

**Comment by tellypresence** [comment]
Please add label "Collada" to this issue

**Comment by tellypresence** [comment]
[`ybot.dae`](https://github.com/assimp/assimp/files/8793399/ybot.dae.zip) loads OK, can this issue be closed?
<img src="https://github.com/user-attachments/assets/6675b30b-ad89-4cae-8cd8-23a943b7e772" width=180 >
